pkg/ffcapi/is_ready.go

```diff
@@ -0,0 +1,24 @@
+// Copyright © 2022 Kaleido, Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ffcapi
+
+import "github.com/hyperledger/firefly-common/pkg/fftypes"
+
+type ReadyResponse struct {
+	Ready             bool             `json:"ready"`
+	DownstreamDetails *fftypes.JSONAny `json:"downstreamDetails"` // The data output from the method call - can be array or object structure
```

Comment:
Idea behind this it can include what the downstream is, how it might be erroring if its down, etc.